templates/poster.html

```diff
@@ -1,4 +1,22 @@
 {% extends "base.html" %}
+
+{% block head %}
+
+{{ super() }}
+<meta name="citation_title" content="{{paper.content.title}}" />
+{% for author in paper.content.authors %}
+<meta name="citation_author" content="{{author}}" />
+{% endfor %}
+<meta name="citation_publication_date" content="{{config.data}}" />
+<meta name="citation_conference_title" content="{{config.name}}" />
+<meta name="citation_abstract" content="{{paper.content.abstract}}" />
+{% for keyword in paper.content.keywords %}
+<meta name="citation_keywords" content="{{keyword}}" />
+{% endfor %}
+<!-- <meta name="citation_pdf_url" content="" /> -->
```

Comment:
Why this is commented out?

Reply:
This could (should?) be replaced by a note in the docs that PDFs would usefully be included here; the exact path by which the PDF url is accessed I suppose will differ per installation

Reply:
Maybe you can add an `if` block?

fwiw: I just learned this default-value magic [here](https://github.com/Mini-Conf/Mini-Conf/pull/40#discussion_r429634739) but maybe this does not apply here.